retailer-oas/retailer.yaml

```diff
@@ -459,63 +714,256 @@ components:
     GeoCoordinates:
       type: object
       properties:
-        lat: 
+        lat:
           type: number
           description: Latitude in decimal degrees
-        lon: 
+        lon:
           type: number
           description: Longitude in decimal degrees
 
     Capacity:
       type: object
       properties:
-        id:
-          type: string
-        total: 
+        total:
           type: number
+          description: Total capacity of the location
```

Comment:
Should we specify the "service type" that the capacity is associated with? We might want DeliveryCapacity and PickingCapacity